.evergreen/functions.yml

```
@@ -169,6 +169,13 @@ functions:
           set -e
           eval $(.evergreen/print-compass-env.sh)
 
+          # With removal of storage-mixin and top level import of 
+          # keytar, we need to rebuild the keytar.
```

I’m not sure that the connection to storage-mixin would be obvious here to future readers

```suggestion
          # Pre-built binaries for keytar do not support older RHEL versions.
          # Since we have started importing keytar from top-level scripts in packages,
          # re-built keytar before the bootstrap process so that it is always usable.
```

That being said, if you want to go back to the previous approach and dynamically load keytar at runtime, I don’t think that that’s a bad choice either.

---

I was thinking about that approach as well and was unsure. As we will be removing `keytar` now, I think we can leave it as is and probably remove at that point.